cmd/dmsg-discovery/internal/api/api.go

```diff
@@ -26,6 +26,9 @@ var log = logging.MustGetLogger("dmsg-discovery")
 
 var json = jsoniter.ConfigFastest
 
+// WhitelistPKs store whitelisted pks of network monitor
+var WhitelistPKs = make(map[string]bool)
```

Comment:
Since this same var is used in `SD` and also in `skywire-services` we can maybe just use it from dmsg to move it to `skywire-utilities`.

Reply:
Done